import { catchErrorAsync } from "./catchErrorAsync";

export default catchErrorAsync;